pom.xml

```diff
@@ -69,9 +70,23 @@
             <url>https://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+    <dependencyManagement>
```

Comment:
Why? We are already reusing the property. And in my experience, `dependencyManagement` does not in fact work to align versions of artifacts with different classifiers.